api_v3/lib/types/filters/base/KalturaUserEntryBaseFilter.php

```diff
@@ -74,7 +74,7 @@ public function getOrderByMap()
 	public $entryIdNotIn;
 
 	/**
-	 * @var int
+	 * @var string
```

Comment:
Noam, this class is auto-generated, please revert all manual changes and apply them elsewhere.